src/opm/output/eclipse/AggregateConnectionData.cpp

```diff
@@ -253,16 +253,19 @@ namespace {
             if (Q.has(R::oil)) {
                 xConn[Ix::OilRate] =
                     - units.from_si(M::liquid_surface_rate, Q.get(R::oil));
+                xConn[17] = xConn[Ix::OilRate];
```

Comment:
The magic numer 17, 18 and 19 need to get a name in the `Ix` namespace.

Reply:
This part is taken out and moved to a separate pull request, where the numbers have got a name in the Ix namespace.